weltyt.py

```diff
@@ -122,7 +122,16 @@ def buy_lands(lands):
     },
 }
 
+commands = {"Help":"Explain all commands", "show_my_resources": "Shows you what resources you have exactly in this time.", "buy": "shows what is possible to buy in this round.", "Plants": "Opens posibility of buying plants.", "Lands": "Opens posibility of buying lands."}
+
+listva = ['Choose what would you like to do. If you want check your resources write "show_my_resources" please. If you want go directly to buying, write "buy". If you want avoid this step write "no" please.\n',
```

Comment:
Why it can't be a dictionary? Why the strange name?

`messages["basic_explanation"]` is much more readable than `listva[0]` - the second one tells us nothing if we don't look up what exactly `listva` is.